pootle/apps/accounts/models.py

```diff
@@ -60,6 +60,10 @@ class User(AbstractBaseUser):
                            _('Enter a valid username.'),
                            'invalid')
         ],
+        # Remove when fixed in Django: https://code.djangoproject.com/ticket/27898#ticket
```

Comment:
That's not true, we always want a human readable error here I'd assume.  The Django issue is about the interpolation which we will no longer use.